.github/workflows/validate.yml

```diff
@@ -0,0 +1,49 @@
+name: validate
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+  pull_request:
+    branches:
+      - master
+      - main
+
+jobs:
+  linters:
+    permissions:
+      contents: read      # for actions/checkout to fetch code
+      pull-requests: read # for golangci/golangci-lint-action to fetch pull requests
+    timeout-minutes: 10
+
+    strategy:
+      matrix:
+        go-version: [1.19.x]
```

Comment:
Linting is only testing on 1.19, as some changes (gofmt etc) may differ between versions of go.

Reply:
Can you add ☝️ as an inline code comment saying "only lint using the latest go version because..."?

Reply:
Yup, makes sense; I'll add